src/main/java/com/codegym/jrugotom5/service/AdvertService.java

```diff
@@ -30,4 +35,21 @@ public List<AdvertDTO> getAdvertsByDateRange(LocalDate from, LocalDate to) {
                 .map(advert -> modelMapper.map(advert, AdvertDTO.class))
                 .collect(Collectors.toList());
     }
+
+    public AdvertDTO update(AdvertDTO advertDTO) {
```

Comment:
Why did you use the same DTO for the update (incoming operation) and to represent it to the client (outgoing).
I assume you need to use a different (INFO) DTO.
Check the main. There may already be convenient DTOs for this purpose.

Reply:
> Why did you use the same DTO for the update (incoming operation) and to represent it to the client (outgoing). I assume you need to use a different (INFO) DTO. Check the main. There may already be convenient DTOs for this purpose.

agree, AdvertFullInfoDTO suits this purpose well